.drone.yml

```diff
@@ -4,40 +4,41 @@ workspace:
 
 services:
   web:
-    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:7.3-ci}
+    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:8.0-ci}
     environment:
       - DOCUMENT_ROOT=/test/oe-poetry-client
 
 pipeline:
   composer-install:
     group: prepare
-    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:7.3-ci}
+    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:8.0-ci}
     volumes:
       - /cache:/cache
     commands:
       - composer update ${COMPOSER_BOUNDARY}
 
   grumphp:
     group: test
-    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:7.3-ci}
+    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:8.0-ci}
     commands:
       - ./vendor/bin/grumphp run
 
   phpunit:
     group: test
-    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:7.3-ci}
+    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:8.0-ci}
     commands:
       - ./vendor/bin/phpunit
 
   phpspec:
     group: test
-    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:7.3-ci}
+    image: ${IMAGE_PHP=registry.fpfis.eu/fpfis/httpd-php:8.0-ci}
     commands:
       - ./vendor/bin/phpspec run
 
 matrix:
   IMAGE_PHP:
-    - registry.fpfis.eu/fpfis/httpd-php:7.3-ci
+    - registry.fpfis.eu/fpfis/httpd-php:8.0-ci
+    - registry.fpfis.eu/fpfis/httpd-php:7.4-ci
```

Comment:
This is not how the matrix is supposed to work. Fix the matrix by providing only the php version instead of the image.
Also, the composer matrix needs to be fixed as well:
![Uploading Screenshot 2022-02-03 at 13.38.13.png…]()